terraform/vmBaseImage.tf

```
@@ -1,18 +1,5 @@
-# Create subnet
-resource "azurerm_subnet" "subnet_server" {
-  name                 = "serverSubnet"
-  resource_group_name  = azurerm_resource_group.myterraformgroup.name
-  virtual_network_name = azurerm_virtual_network.myterraformnetwork.name
-  address_prefixes     = ["10.0.1.0/24"]
-
-  depends_on = [
-    azurerm_resource_group.myterraformgroup,
-    azurerm_virtual_network.myterraformnetwork
-  ]
-}
-
 # Create Network Security Group
-resource "azurerm_network_security_group" "myterraformnsg" {
+resource "azurerm_network_security_group" "nsg_baseImage" {
```

nit: mixed case, either
```suggestion
resource "azurerm_network_security_group" "nsgBaseImage" {
```
or
```suggestion
resource "azurerm_network_security_group" "nsg_base_image" {
```

---

have definitely neglected naming conventions for this repo. I'll change them as I catch them but we definitely need a separate backlog item to clean this stuff up if we plan to use it for anything other than upskilling.